test/NameRegistry.upgrade.t.sol

```diff
@@ -108,6 +139,7 @@ contract NameRegistryV2 is
     Initializable,
     ERC721Upgradeable,
     ERC2771ContextUpgradeable,
+    AccessControlUpgradeable,
```

Comment:
good catch, though i think OwnableUpgradeable in the next line can be removed

Reply:
Done 👍